src/components/FoldableSection/SectionsWithFoldableText.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Text, Heading, Card, CardHeader, CardBody, Box, BoxProps } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import FoldableText from './FoldableText'
+
+interface Props extends BoxProps {
+  header: string
+  config: { title: string; description: string[] }[]
+}
+
+const SectionsWithFoldableText: React.FC<Props> = ({ header, config, ...props }) => {
+  const { t } = useTranslation()
+
+  return (
+    <Box maxWidth="888px" {...props}>
+      <Card>
+        <CardHeader>
+          <Heading scale="lg" color="secondary">
+            {t(header)}
```

Comment:
We should not automatically translate and let the consumer provide the translation.